app/models/base_item.rb

```diff
@@ -28,5 +28,23 @@ class BaseItem < ApplicationRecord
   def to_h
     { partner_key: partner_key, name: name }
   end
+
+  def self.seed_items
```

Comment:
Hmm... this is really only used by test code. I don't think it belongs in the actual model class.

Reply:
Moved this to `rails_helper` not sure if that is the best place for it but seemed logical since it is used only in testing.